hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -53,8 +53,8 @@
 
   enum AllowedBucketLayouts {FILE_SYSTEM_OPTIMIZED, OBJECT_STORE}
 
-  @Option(names = { "--type", "-t" },
-      description = "Allowed Bucket Types: ${COMPLETION-CANDIDATES}",
+  @Option(names = { "--bucketlayout", "-l" },
```

Comment:
Can we simplify `bucketlayout` to `layout`?  The command name `... bucket create` implies the layout is for the bucket.

Also, should we keep `--type`, `-t` as alias for compatibility?

Reply:
Thanks @adoroszlai for the review.

Sure, I will change the name to` "layout"`

>Also, should we keep --type, -t as alias for compatibility?

Since the bucket layout feature is not formally released yet , can we skip the compatibility aspect ?

Reply:
> Since the bucket layout feature is not formally released yet , can we skip the compatibility aspect ?

I don't think it's necessary then.

Reply:
Thanks for the reply, then I will update the patch with the name `"layout"`

Reply:
Done!